packages/chain-helpers/src/blockchain/Blockchain.ts

```diff
@@ -1,7 +1,7 @@
 /**
  * Blockchain bridges that connects the SDK and the KILT Blockchain.
  *
- * Communicates with the chain via WebSockets and can [[listenToBlocks]]. It exposes the [[submitTx]] function that performs a transaction.
+ * Communicates with the chain via WebSockets and can [[listenToBlocks]]. It exposes the [[signTx]] function that performs the necessary tx signing.
```

Comment:
I'd say submission is the more relevant use case? Why should we only mention signing here?

Reply:
The Blockchain Class doesn't really expose any submission function, the api is only used for signing.
BlockchainUtils handles dispatching.